lib/lookup.json

```diff
@@ -256,7 +260,8 @@
     "yarn": true,
     "scripts": ["remove-examples", "build:js", "test-ci-partial"],
     "envVar": { "CI": true },
-    "skip": ["aix", "s390x", "ppc", "darwin", "win32"],
+    "skip": ["aix", "s390x", "ppc", "darwin", "win32", true],
```

Comment:
🙁 since when?

Reply:
Not sure. At least one month: https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/2768/#showFailuresLink

Reply:
worth trying to increase the timeout? FWIW our own CI (ubuntu, GH actions) completes in less than 10 minutes: https://github.com/facebook/jest/runs/4137520913?check_suite_focus=true

Reply:
30 minutes is already 1/10th of a full CITM run. I would not like to increase the timeout beyond that.

Reply:
Are there multiple cores available on Jenkins? If so we can run tests in parallel (that made our own CI about half its execution time)

Reply:
I think we already parallelize the list of modules to test.

Reply:
Possibly related? https://github.com/facebook/jest/issues/11956